system/events/mumble.py

```diff
@@ -396,6 +396,62 @@ class UserRecordingToggle(UserStateToggleEvent):
     pass
 
 
+class UserStats(MumbleEvent):
+    """
+    User stats updated
+
+    user: User whose stats have been updated
+    """
+
+    def __init__(self, caller, user):
+        """
+        Initialise the event object.
+        """
+
+        self.user = user
+
+        super(UserStats, self).__init__(caller)
+
+
+class UserRegisteredEvent(MumbleEvent):
```

Comment:
`UserJoined`, `UserMoved`, `ChannelCreated`, `ChannelLinked`, etc.

`User...Toggle` and `UserRemove` go against this, but they seem in the minority.